docs/src/api/class-locatorassertions.md

```diff
@@ -202,7 +202,7 @@ The opposite of [`method: LocatorAssertions.toContainText`].
 
 ### param: LocatorAssertions.NotToContainText.expected
 * since: v1.18
-- `expected` <[string]|[RegExp]|[Array]<[string]|[RegExp]>>
+- `expected` <[string]|[RegExp]|[Array]<[string]>|[Array]<[RegExp]>|[Array]<[string]|[RegExp]>>
```

Comment:
How does this affect the ports? Does it make them better or worse? I understand that this makes TS typing unnecessary more complex (i.e. makes it a bit worse for the vast majority of our users)